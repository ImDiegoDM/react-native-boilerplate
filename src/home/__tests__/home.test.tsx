import React from 'react';
import { Home } from '../home';
import renderer from 'react-test-renderer';

it('Deve renderizar o componente Home corretamente',()=>{
  const tree = renderer.create(<Home/>).toJSON();
  expect(tree).toMatchSnapshot();
});