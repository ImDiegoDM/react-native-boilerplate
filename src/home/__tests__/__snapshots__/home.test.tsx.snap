// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar o componente Home corretamente 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Home
  </Text>
</View>
`;
